add additional python libraries

master-caster opened the pull request.

master-caster commented:
add python libraries to the images so that ansible can use the additional modules

osfrickler commented on ansible/zuul.yaml:
```diff
@@ -175,6 +175,8 @@
                 - pkg-map
                 - infra-package-needs
                 - zuul-worker
+                - dib-python
```
This only installs a symlink during the image build, I don't see where this would be needed?

osfrickler commented on ansible/zuul.yaml:
```diff
@@ -175,6 +175,8 @@
                 - infra-package-needs
                 - zuul-worker
+                - dib-python
+                - ensure-venv
```
This is already a dependency of infra-package-needs, so kind of redundant. Doesn't hurt to make it explicit, but I'd place it in front of that element, then.

master-caster commented:
@osfrickler thx for review. As shown on https://docs.openstack.org/diskimage-builder/latest/elements/dib-python/README.html
i will remove `dib-python` because it is deprecated and I'm not sure what it was needed for.

Since `ensure-venv` is a dependency for `infra-package-needs` I removed explicit declaration.
Based on the dependencies of `infra-package-needs` I also removed `openssh-server` and `package-installs` since
they should be duplicate too.

I close this PR and create a new one for the new changes.
